src/diffusers/loaders/lora_base.py

```diff
@@ -214,9 +215,18 @@ def _optionally_disable_offloading(cls, _pipeline):
         is_model_cpu_offload = False
         is_sequential_cpu_offload = False
 
+        def model_has_device_map(model):
+            if not is_accelerate_available() or is_accelerate_version("<", "0.14.0"):
+                return False
+            return hasattr(model, "hf_device_map") and model.hf_device_map is not None
```

Comment:
We can just check for the `hf_device_map` attribute. We don't really need to check for accelerate no ?

Reply:
Indeed. I included it just to avoid any edge cases, following what we do in methods like 
https://github.com/huggingface/diffusers/blob/28f9d84549c0b1d24ef00d69a4c723f3a11cffb6/src/diffusers/pipelines/pipeline_utils.py#L391